Kumi.Game/Online/API/IAPIConnectionProvider.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections;
+using Kumi.Game.Online.API.Users;
+using Kumi.Game.Online.Server;
+using osu.Framework.Bindables;
+
+namespace Kumi.Game.Online.API;
+
+/// <summary>
+/// An object that represents a connection to the API.
+/// </summary>
+public interface IAPIConnectionProvider
```

Comment:
Where's the actual implementation for this interface that's not the dummy API connection?

Reply:
Waiting for whenever I push to [kikoyudev/kumi-api](https://github.com/kikuyodev/kumi-api).